File: C#/1036.cs
using System;

class URI
{

    static void Main(string[] args)
    {
        double a, b, c, r1, r2, delta;
        string[] linha1 = Console.ReadLine().Split(' ');
        a = Double.Parse(linha1[0]);
        b = Double.Parse(linha1[1]);
        c = Double.Parse(linha1[2]);
        delta = (b * b) - (4.0 * a * c);
        if (0 > delta || (2.0 * a) == 0.0)
            Console.WriteLine("Impossivel calcular");
        else
        {
            r1 = (-b + Math.Sqrt(delta)) / (2.0 * a);
            r2 = (-b - Math.Sqrt(delta)) / (2.0 * a);
            Console.WriteLine("R1 = {0:0.00000}", r1);
            Console.WriteLine("R2 = {0:0.00000}", r2);

        }
    }

}

File: C++/1036.cpp
#include <iostream>
#include <string>
#include <limits>
#include <cmath>
#include <iomanip>

using namespace std;

int main(){

    double a, b, c;

    cin >> a >> b >> c;

    if(a == 0 || ((b*b)-(4*a*c)) <= 0)
    {
        cout << "Impossivel calcular\n";
        return 0;
    }
    double r1 = (-b+(sqrt((b*b)-(4*a*c))))/(2*a);
    double r2 = (-b-(sqrt((b*b)-(4*a*c))))/(2*a);
    cout << fixed << setprecision(5) << "R1 = " << r1 << "\n";
    cout << fixed << setprecision(5) << "R2 = " << r2 << "\n"; 

}
